src/pages/docs/creators/javascript/examples/tools/requestor-uninstall.md

```diff
@@ -0,0 +1,124 @@
+---
+title: How to remove Yagna?
+description: Step-by-step guide to removing Yagna components, configs, and data for Requestors.
+type: noindex
```

Comment:
Noindex should probably not be here. We want that to be searchable

Reply:
I will edit that as it doesn't allow me to search on the preview. I think we should always check how our search looks when adding new articles. They should be easy to find using proper keywords.